docs/guides/getting-started/setup/_fragments/_intro.mdx

```diff
@@ -11,18 +9,4 @@ Tauri is a framework to build desktop applications with any frontend framework a
 1. Rust binary that creates the windows and exposes native functionality to those windows
 2. Frontend of your choice that produces the user interface inside the window
 
-In the following, we will first scaffold the frontend, set up the Rust project, and lastly show you how to communicate between the two.
-
-:::tip `create-tauri-app`
-
-The easiest way to scaffold a new project is the [`create-tauri-app`] utility. It provides opiniated templates for vanilla HTML/CSS/JavaScript and many frontend frameworks like React, Svelte, and Yew.
-
-<CreateTauriApp />
-
-Please note that you do not need to follow the rest of this guide if you use `create-tauri-app`, but we still recommend reading it to understand the setup.
-
-:::
-
-[cargo]: https://doc.rust-lang.org/cargo/
-[prerequisites]: ../../prerequisites.md
```

Comment:
Need to keep this link in here so that the link in line 3 isn't broken

Reply:
@lorenzolewis Thanks, I left the `prerequisites` link here, but I removed the `cargo` link on `_create-tauri-app-guide.mdx` because I think the link isn't used on the page.